config.lua

```diff
@@ -1,5 +1,5 @@
 cfg = {
-    position = 'top-right', --Changes the position of the notifications
+	position = 'top-right', --Changes the position of the notifications
 	maxNotifications = 0, --Max notifications to show on screen (0 indicates no limit)
```

Comment:
Inconsistent indentation, can we change this to 2 space indentation.

Reply:
I'll change it. This was changed as shown on the UPD branch.